content/millennium/r4/scheduling/appointment.md

```diff
@@ -199,6 +241,8 @@ In addition, the following errors may be returned:
 * Updating an Appointment resource without sending the `If-Match` header will result in a `412 Precondition Failed` response.
 * Updating an Appointment resource which is currently being modified will result in a `423 Locked` response.
 * If the Appointment resource could not be updated because of an operation that is necessary for the update (eg. encounter association), `424 Failed Dependency` response will be returned.
+* Patching a video visit that has previously been patched for video links will result in a `409 Conflict` response.
```

Comment:
Patching a video visit appointment that has previously been patched for video visit links ...

Reply:
https://github.com/cerner/fhir.cerner.com/pull/294/commits/8d9a3a93693a681c6c079a2ad13ae65e362804c0#diff-1d7df8ceb9f0ae8471323516e037c662R243